Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -141,20 +190,70 @@ final class PledgePaymentPlanOptionView: UIView {
   @objc private func onOptionTapped() {
     self.viewModel.inputs.optionTapped()
   }
+
+  @objc private func onTermsOfUseButtonTapped() {
+    self.viewModel.inputs.termsOfUseTapped()
+  }
+
+  // MARK: - Functions
+
+  private func setupIncrementsStackView(with increments: [PledgePaymentIncrementFormatted]) {
+    var dateLabels: [UILabel] = []
+    increments.forEach { increment in
+      let incrementStackView = UIStackView()
+      applyIncrementStackViewStyle(incrementStackView)
+
+      let titleLabel = UILabel()
+      applyIncrementTitleLabelStyle(titleLabel)
+      titleLabel.text = increment.title
+
+      let detailsStackView = UIStackView()
+      applyIncrementDetailsStackViewStyle(detailsStackView)
+
+      let dateLabel = UILabel()
+      applyIncrementDateLabelStyle(dateLabel)
+      dateLabel.text = increment.scheduledCollection
+      dateLabels.append(dateLabel)
+
+      let amountLabel = UILabel()
+      applyIncrementDateLabelStyle(amountLabel)
+      amountLabel.text = increment.amount
+      amountLabel.setContentCompressionResistancePriority(.required, for: .horizontal)
+
+      detailsStackView.addArrangedSubviews(dateLabel, amountLabel)
+      incrementStackView.addArrangedSubviews(titleLabel, detailsStackView)
+
+      self.paymentIncrementsStackView.addArrangedSubview(incrementStackView)
+    }
+
+    if let firstDateLabel = dateLabels.first {
+      dateLabels.forEach { label in
+        label.widthAnchor.constraint(equalTo: firstDateLabel.widthAnchor).isActive = true
```

Comment:
Intervals dates/amount alignment issue fixe

Explaining the issue: when the increments table mixed dates with one-digit days (`4 Jan 2025`) and two-digit days (`14 Feb 2025`), the `amountLabel` was misaligned. To fix it, a width constraint (all equal width) was added to each dateLabel.

Reply:
Sure, added!